### ISSUE TITLE
Spinner shows in tab bar in Ubuntu

### ISSUE THREAD

**Issue by phase1geo** [phase1geo/Minder#151]
Thank you very much for all your hard work and dedication; 
by the way, there is something you can see it in the picture
Whenever a new tab is added, a spinner appears at the top, stays there all the time.
![2020-06-03_16-47](https://user-images.githubusercontent.com/13304792/83658431-1ac77200-a5ba-11ea-98bc-f4158f231455.png)
The problem has only appeared in Ubuntu, not in elementary os

_Originally posted by @wada3n in https://github.com/phase1geo/Minder/issues/149#issuecomment-638287505_

**Comment by phase1geo** [comment]
I am able to replicate this issue, but only using the 20.04 version of Ubuntu.  I tried it out on the 19.10 version and the spinner is not displayed.

I'm going to hold off fixing this issue (I don't actually understand what is causing the issue, but I don't believe this issue is specific to Minder at this point) until the elementary OS team releases the developer version of the next major OS release which will ship with the Ubuntu 20.04 version of the Gtk library.  I have a feeling that elementary's updates for this release will most likely fix the issue.  I will potentially have other issues that I will need to figure out when the next release is available as well.

**Comment by wada3n** [comment]
By the way, the same issue exists in Outliner, Waiting for the new version of elementary 6.0.
Thank you very much for your great efforts

**Comment by phase1geo** [comment]
Yes, both Minder and Outliner would have the same issue. The tab bar is shared code between the two.

**Comment by FreeTheSmoke** [comment]
Other notes that may help eventual diagnosis-
- I thought maybe the spinner was like an unsaved changes asterix, but it appears regardless of if a new/empty map is opened, or an existing saved one
-  It remains centered, adjusting slightly when the properties tab is opened.
- Ubuntu 20.04
- I realise the above things are not very helpful, I'm really enjoying minder though, so will send donation to supplement :)
